Documentation updates (#45)
MicrosoftDocs/microsoft-authentication-library-dotnet: msal-dotnet-articles/advanced/backup-authentication-system.md

---

> title: Backup authentication system

Reviewer: Maybe this page is a good candidate to move into a more general resiliency page, like High Availability, as a section.

Author: Indeed, but I will defer that to another pull request rather than this one.

description: "The Azure Active Directory backup authentication system enables the caching of credentials processed by the Evolved Security Token Service (ESTS) in order to provide resiliency during outages in Azure AD's authentication services."
---

# Backup authentication system

> The Azure Active Directory backup authentication system enables the caching of credentials processed by the Evolved Security Token Service (ESTS) in order to provide resiliency during outages in Azure AD's authentication services.

Reviewer: If I remember, the docs folks didn't want to use ESTS term - only Azure AD (too technical probably?)

```suggestion
The Azure Active Directory has a backup authentication system that enables the caching of credentials in order to provide resiliency during outages in Azure AD's authentication services.
```
